scripting/get5.sp

```diff
@@ -1089,9 +1089,23 @@ static Action Command_LoadMatchUrl(int client, int args) {
     ReplyToCommand(client,
                    "Cannot load matches from a url without the SteamWorks extension running");
   } else {
-    char arg[PLATFORM_MAX_PATH];
-    if (args >= 1 && GetCmdArgString(arg, sizeof(arg))) {
-      if (!LoadMatchFromUrl(arg)) {
+    char url[PLATFORM_MAX_PATH];
+    if (args >= 1 && GetCmdArg(1, url, sizeof(url))) {
+      ArrayList headerNames = new ArrayList(PLATFORM_MAX_PATH);
```

Comment:
These array lists can be declared here (`ArrayList headerNames;`), but you don't need to assign to them (`new`) unless they are needed below, and you must also `delete` both of these after you call `LoadMatchFromURL` below.